reco_utils/recommender/sar/sar_singlenode.py

```diff
@@ -282,6 +286,18 @@ def score(self, test, remove_seen=False):
         if isinstance(test_scores, sparse.spmatrix):
             test_scores = test_scores.toarray()
 
+        if normalized:
+            if remove_seen:
```

Comment:
`remove_seen` only applies when `normalized` is `True`?

Reply:
this changed, now they are more clearly independent